addons/payment_test/README.md

```diff
@@ -0,0 +1,15 @@
+# Wire Transfer
+
+## Technical details
+
+This module does not integrate with an API and, instead, uses the payment engine to immediately mark
+Wire Transfer transactions as 'pending' to display the 'pending message' containing payment
+instructions on the confirmation page.
+
+## Supported features
+
+- Direct payment flow
+
+## Testing instructions
+
+Wire Transfer doesn't have a test mode.
```

Comment:
It is a bit confusing as you can put this provider in test mode, then it has a test mode. Maybe it is better to say that it does not require the testing details?

Reply:
Indeed, I stated that test and live more are identical.

Reply:
Also, I wrote the README of `payment_transfer` in `payment_test` :man_facepalming:

Added a README for `payment_test` too.